### ISSUE TITLE
Configure via ENV var: listener.security.protocol.map 

### ISSUE THREAD

**Issue by gintautassulskus** [bitnami/bitnami-docker-kafka#26]
It would be great if the `listener.security.protocol.map` property could be configured via ENV variable, e.g.: KAFKA_LISTENER_SECURITY_PROTOCOL_MAP.

Just to double check, am I correct saying that at the moment the mapping between the ENV variables and the configuration properties is hardcoded? 
If yes, would it be sensible to create a dynamic ENV->property parser that would parse all ENV variables starting with `KAFKA_` or `KAFKA_CFG_` to the configuration keys, e.g. KAFKA_SOME_PROPERTY as `some.property`?

**Comment by gintautassulskus** [comment]
Just noticed, this is the same issue as this one: https://github.com/bitnami/bitnami-docker-kafka/issues/12

**Comment by tompizmor** [comment]
Hello @gintautassulskus,

We have just added the KAFKA_LISTENER_SECURITY_PROTOCOL_MAP env var to the kafka image. The [configuration section](https://github.com/bitnami/bitnami-docker-kafka#configuration) in the README has been also updated.

> Just to double check, am I correct saying that at the moment the mapping between the ENV variables and the configuration properties is hardcoded?

Yes. Right now you can see the values in [this file](https://github.com/bitnami/bitnami-docker-kafka/blob/master/2/debian-9/rootfs/kafka-inputs.json)

> If yes, would it be sensible to create a dynamic ENV->property parser that would parse all ENV variables starting with KAFKA_ or KAFKA_CFG_ to the configuration keys

Agree that would be a nice to have feature.  As it is not an issue in the image there are no immediate plans to work on it. Nevertheless, it will be considered for future application versions

**Comment by gintautassulskus** [comment]
Thanks, @tompizmor, it works now.

**Comment by tompizmor** [comment]
@gintautassulskus I just wanted to let you know that we have created the dynamic ENV->property parser that you were suggesting in your first comment. 

Now, each env var starting by KAFKA_CFG is translated into a server.properties configuration entry.

The [configuration](https://github.com/bitnami/bitnami-docker-kafka#configuration) section in the README has been updated to document the use of this new feature.
